ground/gcs/src/plugins/rawhid/rawhidplugin.h

```diff
@@ -95,7 +92,7 @@ class RAWHID_EXPORT RawHIDPlugin
     virtual bool initialize(const QStringList &arguments, QString *error_message);
     virtual void extensionsInitialized();
 private:
-    RawHIDConnection *hidConnection;
+    RawHIDConnection* hidConnection;
```

Comment:
Qt coding style = * with the symbol according to https://wiki.qt.io/Qt_Coding_Style